config/ember-try.js

```diff
@@ -46,8 +46,25 @@ module.exports = function() {
             }
           }
         },
+        {
+          name: 'ember-lts-3.4',
+          npm: {
+            devDependencies: {
+              'ember-source': '~3.4.0'
+            }
+          }
+        },
+        {
+          name: 'ember-lts-3.8',
+          npm: {
+            devDependencies: {
+              'ember-source': '~3.8.0'
+            }
+          }
+        },
         {
           name: 'ember-release',
+          allowedToFail: true,
```

Comment:
🤔 Do we want to allow release to fail?  Beta and Canary make sense, but release feels important.

Looking through the logs, it seems to be getting tripped up on `lookup`
https://github.com/poteto/ember-cli-flash/blob/master/tests/helpers/utils/lookup.js#L6

Reply:
we might be able to use the [`getContext()` test helper](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#getcontext) until we refactor it to use `this.owner`

Reply:
🤦‍♂ I was working on the version upgrade and added this props temporarily. Missed removing it. Will get that sorted👍

Reply:
It looks like the lookup util you had pointed @sbatson5 is not the problem. Migrating the integration test code to the latest `ember-qunit` format makes the tests to pass. However, this change is done in #308. Should I try merging this branch into the integration test branch and check if that works?

Reply:
I'd definitely like to get the whole suite passing and supporting ember-release before we release a major version, but I'm happy to do this as a separate PR.  How much effort is it to get it passing with `ember-release` 😅 ?

Reply:
Let's remove this flag and merge, even if the suite is failing (since we'll still be in the same state).  Then we can update https://github.com/poteto/ember-cli-flash/pull/308, since that isn't even running due to the node version

Reply:
Awesome. Sounds good👍. I'll remove the flag.

Reply:
@sbatson5 the flag is now removed